yarn.lock

```diff
@@ -2808,13 +2808,18 @@
   dependencies:
     "@guardian/libs" "1.6.1 - 3.3.0"
 
-"@guardian/discussion-rendering@^10.1.1":
-  version "10.1.1"
-  resolved "https://registry.yarnpkg.com/@guardian/discussion-rendering/-/discussion-rendering-10.1.1.tgz#7613046322253bdcbd18b2a96647911df27fce25"
-  integrity sha512-HHAIkVw+CnJizYESYCm8Dt3rlLVVVmbRMaVqCT1E09uio7Zg5ShXMcv2rjJ/WzsjCGGtBTJ66v5x+dkZqWeZqA==
+"@guardian/discussion-rendering@10.2.0":
+  version "10.2.0"
+  resolved "https://registry.yarnpkg.com/@guardian/discussion-rendering/-/discussion-rendering-10.2.0.tgz#9f0976266b193ef1955d74d0f3584672083f7ed5"
+  integrity sha512-w0s/qNhN3djBsm+ci2X42EafEbAi7sJPof9bgyzP2HxZfWE5IIsiZVQ9G31KvigMROBfcjGrWRVEorP5vDrt8g==
   dependencies:
+    "@rollup/plugin-html" "^0.2.4"
+    "@rollup/plugin-replace" "^4.0.0"
     babel-plugin-emotion "^10.0.33"
     customize-cra "^1.0.0"
+    rollup-plugin-livereload "^2.0.5"
+    rollup-plugin-serve "^1.1.0"
+    rollup-plugin-terser "^7.0.2"
```

Comment:
@mxdvl @bryophyta I'm curious about these new dependencies from `@guardian/discussion-rendering`. I think they came via https://github.com/guardian/discussion-rendering/commit/073da657b8286511a538537ab30022d3b28110df but I wonder if they should be `devDependencies`?

Reply:
Oh yes, most definitely these are `devDependencies`!